datadog-cloudformation-common-python/src/datadog_cloudformation_common/api_clients.py

```diff
@@ -6,14 +6,13 @@
 @contextmanager
 def v1_client(api_key: str, app_key: str, api_url: str, resource_name: str, resource_version: str) -> ApiClient:
     configuration = Configuration(
-        host=api_url,
+        host=api_url or "https://api.datadoghq.com",
```

Comment:
It looks like the schema available to use in resources uses a version of JSON draft 7 (https://docs.aws.amazon.com/cloudformation-cli/latest/userguide/resource-type-schema.html) however it doesn't looks like the python sdk sets a default, so hardcoding the default api URL here instead of in each resource handler call.